ui/src/components/banner/Banner.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+import DismissIcon from 'icons/DismissIcon';
+import CriticalHexagonIcon from 'icons/CriticalHexagonIcon';
+import WarningTriangleIcon from 'icons/WarningTriangleIcon';
+import InfoCircleIcon from 'icons/InfoCircleIcon';
+import { BANNER_TYPES } from 'utils/constants';
+
+import {
+  BannerWrapper,
+  BannerContent,
+  BannerText,
+  BannerTitle,
+  BannerMessage,
+  DismissButton,
+} from 'theme/bannerStyles';
+
+const BannerIcon = ({ type, ...props }) => {
+  switch (type) {
+    case BANNER_TYPES.critical:
+      return <CriticalHexagonIcon {...props} />;
+    case BANNER_TYPES.warning:
+      return <WarningTriangleIcon {...props} />;
+    case BANNER_TYPES.info:
+    default:
+      return <InfoCircleIcon {...props} />;
+  }
+};
+
+const Banner = ({ dismissible = true, message, title, type = BANNER_TYPES.info }) => {
+  const [active, setActive] = useState(true);
```

Comment:
@mistryrn does this mean the banner will open again on every new session/tab? good enough for the urgent banners, but wondering if that was meant "as is" on purpose.

@MitchellShiell for context here, in other apps we've been keeping in localStorage whether you have dismissed the banner, so you don't have to do it every time you visit the site.